app/src/views/Game.vue

```diff
@@ -117,28 +120,17 @@ export default {
     },
     isUserDead() {
       return this.gameState?.statuses[this.userId] == PlayerStatus.Dead;
+    },
+    playerCount() {
+      return this.gameState?.players.length;
+    },
+    deckCount() {
```

Comment:
`playerCount` is the number of players but `deckCount` is not the number of decks ? 😛

Reply:
Replaced with cardsInDeck

Reply:
cardsInDeck actually already existed xD

Reply:
haha !